let a=100;

try {
    let b=100/0;
    console.log(b);
} catch (error) {
    console.log("mathematical error ");
}
